politeiawww/inventory.go

```diff
@@ -136,6 +146,28 @@ func (b *backend) _updateInventoryCountOfPropStatus(status pd.RecordStatusT, old
 	executeUpdate(1, convertPropStatusFromPD(status))
 }
 
+// _updateInventoryCountOfUserProposals updates the count of proposals per user ID
+// Must be called WITH the mutex held
+func (b *backend) _updateCountOfUserProposals(token string) {
```

Comment:
This is obviously broken. This function needs to return an error and you need to validate said error before continuing.